packages/core/src/commands/insertContentAt.ts

```diff
@@ -64,11 +64,6 @@ export const insertContentAt: RawCommands['insertContentAt'] = (position, value,
       },
     })
 
-    // don’t dispatch an empty fragment because this can lead to strange errors
-    if (content.toString() === '<>') {
-      return true
-    }
-
```

Comment:
This was removed because `commands.clearContent` relies on `setContent` to be able to reset the document to the initial schema. This was early exiting and causing a no-op making `clearContent` not actually clear anything.

This is very old code that wasn't very well documented but after some testing, I'm confident in this not "leading to strange errors". `insertContentAt` is much more robust than when this was written.

I tested inserting empty content into a node and a block and it essentially resulted in a no-op.